packages/@cdktf/provider-generator/lib/__tests__/provider.test.ts

```diff
@@ -61,5 +61,47 @@ describe("Provider", () => {
       const snapshot = directorySnapshot(workdir);
       expect(snapshot).toMatchSnapshot();
     });
-  }, 600_000);
+  }, 600_000),
+    it.only("has generated provider that includes static import functions", async () => {
```

Comment:
We should remove the only for CI to run everything
```suggestion
    it("has generated provider that includes static import functions", async () => {
```

Reply:
👍